packages/sdk/config/src/plugin/definitions.ts

```diff
@@ -25,8 +25,8 @@ export const definitions = ({
   env,
 }: ConfigPluginOpts) => {
   const KEYS_TO_FILE = {
-    __CONFIG_DEFAULTS__: configPath ?? resolve(CWD, 'dx.yml'),
-    __CONFIG_ENVS__: envPath ?? resolve(CWD, 'dx-env.yml'),
+    __CONFIG_DEFAULTS__: configPath && configPath.length ? configPath : resolve(CWD, 'dx.yml'),
+    __CONFIG_ENVS__: envPath && envPath.length ? envPath : resolve(CWD, 'dx-env.yml'),
```

Comment:
Environment wouldn't load for me without this.

Reply:
interesting... wonder why that's not come up before. I think it could be shortened to `configPath?.length` but seems reasonable to me

Reply:
Hmm.. I'm guessing it's because I'm using this shell that almost no one else uses (elvish).